tests/e2e/specs/analysis.cy.js

```diff
@@ -265,3 +265,91 @@ describe('Analysis view', () => {
     })
   })
 })
+
+function addView (view) {
+  cy.get('[data-cy=add-view-btn]').click()
+  cy.get(`#toolbar-add-${view}-view`).click()
+    // wait for menu to close
+    .should('not.be.exist')
+}
+
+describe('Filters and Options save state', () => {
+  const numTasks = sortedTasks.length
+  describe('Options save state', () => {
+    it('remembers table and box & whiskers toggle option when switching between workflows', () => {
+      cy.visit('/#/workspace/one')
+      addView('Analysis')
+      cy.get('.c-analysis [data-cy=box-plot-toggle]')
+        .click()
+        .get('.vue-apexcharts')
+        .should('be.visible')
+      // Navigate away
+      cy.visit('/#/')
+      cy.get('.c-dashboard')
+      // Navigate back
+      cy.visit('/#/workspace/one')
+      cy.get('.vue-apexcharts')
+        .should('be.visible')
+    })
+  })
+  describe('Filters save state', () => {
+    it('remembers task name, platform and timings when switching between workflows', () => {
+      cy.visit('/#/workspace/one')
+      addView('Analysis')
+      // Check default options
+      cy
+        .get('.c-analysis table > tbody > tr')
+        .should('have.length', numTasks)
+        .should('be.visible')
+      cy
+        .get('td')
+        .contains('30')
+        .should('be.visible')
```

Comment:
This does not appear to contribute to the test in a meaningful way, as we are not checking later on that it still is or no longer is visible.
```suggestion
```